src/main/java/io/anserini/index/IndexCollection.java

```diff
@@ -229,8 +229,10 @@ public void run() {
               inputFile.getFileName().toString() + ": error iterating through segment.");
         }
 
-        LOG.info(inputFile.getParent().getFileName().toString() + File.separator +
-            inputFile.getFileName().toString() + ": " + cnt + " docs added.");
+        if (verbose) {
```

Comment:
Instead of marking up the code with if statements for verbosity, maybe move the finer-grained logging to the DEBUG level and have a `-verbose` flag that sets the logger to this level?

Reply:
Good call. Will do.